techniques/css/C32.html

```diff
@@ -236,29 +232,29 @@ <h2>Related Resources</h2>
 
 
 <section id="related">
-      <h2>Related Techniques</h2>
+			<h2>関連テクニック</h2>
       <ul>
         <li><a href="../css/C31">C31: Using CSS Flexbox to reflow content</a></li>
       </ul>
     </section>
 <section id="tests">
-      <h2>Tests</h2>
+      <h2>検証</h2>
       <section class="test-procedure" id="procedure">
-        <h3>Procedure</h3>
+        <h3>手順</h3>
         <ol>
-          <li>Display the web page in a user agent capable of 400% zoom and set the viewport dimensions (in CSS pixels) to 1280 wide and 1024 high.</li>
-          <li>Zoom in by 400%.</li>
-          <li>For content read horizontally, check that all content and functionality is available without horizontal scrolling.</li>
-          <li>For content read vertically, check that all content and functionality is available without vertical scrolling.</li>
+          <li>400 ％のズームが可能なユーザエージェントでウェブページを表示し、ビューポートサイズ (CSS ピクセル単位) を幅 1280、高さ 1024 に設定する。</li>
+          <li>400 ％にズームする。</li>
+          <li>水平方向に読むコンテンツの場合、すべてのコンテンツと機能が横スクロールせずに利用できることを確認する。</li>
+          <li>垂直方向に読むコンテンツの場合、すべてのコンテンツと機能が縦スクロールせずに利用できることを確認する。</li>
```

Comment:
check that ... は他のテクニックと合わせて「チェックする」としました。
```suggestion
          <li>垂直方向に読むコンテンツの場合、すべてのコンテンツと機能が縦スクロールせずに利用できることをチェックする。</li>
```